set.mm

```diff
@@ -507309,6 +507630,23 @@ codes is an increasing chain (with respect to inclusion).  (Contributed
       NUIUABJKUGUHOPQFERT $.
   $}
 
+  ${
+    $d A a $.  $d B a $.  $d M a $.  $d V a $.
+    $( The wff ` ( A e. B -/\ B e. A ) ` encoded as ` ( ( A e.g B ) `
+       ` |g ( B e.g A ) ) ` is true in any model ` M ` , i.e., if for every
```

Comment:
```suggestion
       ` |g ( B e.g A ) ) ` is true in any model ` M ` , i.e., for every
```

Reply:
The i.e. refers to "in any model", and this is equivalent with "if for every ...". Therefore, the "if" is correct here (or may even be replaced by "iff", see comment for ~prv).

Reply:
I would remove the "i.e." clause altogether, since the comment is sufficiently clear without it.

Reply:
Done.